src/graphql/create-user-with-email.ts

```diff
@@ -31,18 +31,22 @@ const createUserWithEmail = {
     if (!args.username) {
       throw `Username can't be an empty string`;
     }
+
+    // TODO: Make a proper fix
+    args.username = args.username.toLowerCase();
+
     const res = await getByUsernameOrEmailOrPublicKey(
       args.username,
       args.email,
       args.publicKey
     );
-    if (res && res._highlightResult.hasOwnProperty('email')) {
+    if (res && res.email === args.email) {
```

Comment:
not sure why this changed

Reply:
I think there were changes merged only to the master, and now they got applied to staging as well.